jannovar-core/src/test/java/de/charite/compbio/jannovar/annotation/builders/DeletionAnnotationBuilderTest.java

```diff
@@ -1221,7 +1221,7 @@ public void testRealWorldCase_uc011mcs_() throws InvalidGenomeVariant {
 		Assert.assertEquals(infoForward.getAccession(), annotation1.getTranscript().getAccession());
 		Assert.assertEquals(12, annotation1.getAnnoLoc().getRank());
 		Assert.assertEquals("1068_1071del", annotation1.getCDSNTChange().toHGVSString());
-		Assert.assertEquals("(Glu358del)", annotation1.getProteinChange().toHGVSString(AminoAcidCode.THREE_LETTER));
+		Assert.assertEquals("(Glu357Lysfs*?)", annotation1.getProteinChange().toHGVSString(AminoAcidCode.THREE_LETTER));
```

Comment:
I wasn't able to verify _this specific_ variant, just changed this to the output now generated by the fixed method. Maybe the best strategy here would be to use a 'known valid' protein change description as ground truth (e.g. from ClinVar) and add another test case.